File: Domovoy/Models/SmartHomeDeviceActionLogEntry.cs
namespace Domovoy.Models;

public class SmartHomeDeviceActionLogEntry
{
    public int Id { get; set; }
    
    public bool Perfomed { get; set; }
    public DateTime DateTime { get; set; }
    public string ActionId { get; set; }
    
    public SmartHomeDevice SmartHomeDevice { get; set; }
}

public class SmartHomeDeviceActionLogEntryDTO
{
    public int Id { get; set; }
    
    public bool Perfomed { get; set; }
    public DateTime DateTime { get; set; }
    public string ActionId { get; set; }
}

File: Domovoy/ClientApp/src/api/models/SmartHomeDeviceActionLogEntry.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { SmartHomeDevice } from './SmartHomeDevice';

export type SmartHomeDeviceActionLogEntry = {
    id?: number;
    perfomed?: boolean;
    dateTime?: string;
    actionId?: string | null;
    smartHomeDevice?: SmartHomeDevice;
};